rstudio/rstudio
# Move git commit sign checkbox

### Intent

Addresses [The "sign commit" button should be opt-in #14559](https://github.com/rstudio/rstudio/issues/14559)

### Approach

Moved the "sign commit" checkbox from the commit dialog to the Global Options Git/SVN dialog, and updated docs.

### Automated Tests

None

### QA Notes

Quick sanity check that selecting the checkbox causes commits via the UI to include `--gpg-sign` option. You can see this if you scroll the results to the right:

<img width="657" alt="screenshot of message shown after a git commit" src="https://github.com/rstudio/rstudio/assets/10569626/3b8dc256-442d-4ff7-8ccb-40910eba012a">

### Documentation

User Guide is updated with this PR.

### Checklist
